src/test/scala/mesosphere/marathon/api/v2/json/FormatsTest.scala

```diff
@@ -0,0 +1,110 @@
+package mesosphere.marathon.api.v2.json
+
+import java.util.UUID
+
+import mesosphere.marathon.MarathonSpec
+import mesosphere.marathon.api.v2.GroupUpdate
+import mesosphere.marathon.state.{ Timestamp, AppDefinition, Group }
+import play.api.libs.json.{ JsNull, JsValue, Format, Json }
+import mesosphere.marathon.state.PathId._
+import org.scalatest.Matchers._
+
+class FormatsTest extends MarathonSpec {
+  import Formats._
+
+  test("Can read GroupUpdate json") {
```

Comment:
You extended the tests majorly. That's great!

But why did you choose a more general test name? It really only tests the GroupFormat, does it?

I actually do like more specific test suits that don't get too long.